### ISSUE TITLE
New issue

### ISSUE THREAD

**Issue by ehabdevel** [ehabdevel/new-ste#1]
This is a new issue for the new bot

**Comment by ehabdevel** [comment]
/depends on #1 and #2 are resolved first

**Comment by ehabdevel** [comment]
/depends on #1 and #2 are resolved first

**Comment by ehabdevel** [comment]
/depends on #1 and #2 are resolved first

**Comment by ehabdevel** [comment]
/depends on #1 and #2 are resolved first

**Comment by ehabdevel** [comment]
/depends on #1 and #2 are resolved first
